docroot/modules/custom/va_gov_vet_center/src/EventSubscriber/EntityEventSubscriber.php

```diff
@@ -218,8 +242,9 @@ public function requireRevisionMessage(array &$form, FormStateInterface &$form_s
   public static function getSubscribedEvents(): array {
     return [
       HookEventDispatcherInterface::FORM_ALTER => 'formAlter',
-      // React on Vet center locations list edit form.
       'hook_event_dispatcher.form_node_vet_center_locations_list_edit_form.alter' => 'alterVetCenterLocationsListNodeEditForm',
+      'hook_event_dispatcher.form_node_vet_center_edit_form.alter' => 'unsetNameFieldForNonAdmins',
+      'hook_event_dispatcher.form_node_vet_center_form.alter' => 'unsetNameFieldForNonAdmins',
```

Comment:
This pattern prevents these form.alter from doing anything else.   They can only do `unsetNameFieldForNonAdmins`
A better pattern would be something like
```suggestion
      'hook_event_dispatcher.form_node_vet_center_form.alter' => vetcenterFormAlter',
```

Then inside that function it would call `unsetNameFieldForNonAdmins` and any other methods we need to call.

Reply:
Yep, agreed. Got lazy - updated, @swirtSJW :+1: